README.md

```diff
@@ -35,15 +35,31 @@ your needs. The format is shown in the example. The simplest form would be:
 ```
 
 *Important Note:* You have to restart foreman in order to apply changes in
-`default_hostgroup.yaml`!
+`foreman_default_hostgroup.yaml`!
```

Comment:
Actually, furthermore, you'll need to update the gemspec and possible the rpm/deb packaging too - is there a good reason behind this rename?